Lib/fontTools/ttLib/ttGlyphSet.py

```diff
@@ -488,3 +488,35 @@ def draw(self, pen):
         factor = self.glyphset.factor
 
         replayRecording(lerpRecordings(recording1.value, recording2.value, factor), pen)
+
+
+if __name__ == "__main__":
+
+    import sys
+
+    # Load font and draw all glyphs
+    from fontTools.ttLib import TTFont
+    from fontTools.pens.basePen import NullPen
+
+    if len(sys.argv) < 2:
+        print("Usage: ttGlyphSet.py [-y fontNumber] [-n numGlyphs] fontfile.ttf")
+        sys.exit(1)
```

Comment:
You should not need this block now that you are using argparse.

Reply:
Thanks.